commands/commands.go

```diff
@@ -55,6 +57,13 @@ var (
 				fileLogger, _ := lumber.NewAppendLogger(filepath.ToSlash(filepath.Join(config.GlobalDir(), "nanobox.log")))
 				lumber.SetLogger(fileLogger)
 
+			} else {
+				if !strings.Contains(ccmd.CommandPath(), "server")  && util.IsPrivileged() {
```

Comment:
Maybe you should also check for CI mode too since CI systems often run as `root`